package.json

```diff
@@ -226,5 +233,41 @@
       "protractor-cucumber-framework",
       "postcss-cssnext"
     ]
+  },
+  "jest": {
+    "modulePaths": ["<rootDir>/src/components"],
+    "testMatch": [
```

Comment:
the `testMatch` could contain commented out a list of all the test files, so that we can easily see the progress of the migration. Enabling a new test in jest would be just uncommenting it here.